pkg/retry/retry.go

```diff
@@ -35,6 +35,13 @@ const (
 	errCallingGitHub       = "Failed to call GitHub."
 )
 
+// EventIdentifier represents the required information used by the retry
+// service for handling a GitHub event.
+type EventIdentifier struct {
+	eventID *int64
```

Comment:
Why a pointer?

Reply:
the github library im using is using a *int64 so i was just matching them.

Reply:
ill remove it though